jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/udp/spat/SpatReceiver.java

```diff
@@ -0,0 +1,129 @@
+package us.dot.its.jpo.ode.udp.spat;
+
+import java.net.DatagramPacket;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+import java.time.format.DateTimeFormatter;
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import org.apache.tomcat.util.buf.HexUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import us.dot.its.jpo.ode.coder.StringPublisher;
+import us.dot.its.jpo.ode.OdeProperties;
+import us.dot.its.jpo.ode.udp.AbstractUdpReceiverPublisher;
+
+public class SpatReceiver extends AbstractUdpReceiverPublisher {
+
+   private static Logger logger = LoggerFactory.getLogger(SpatReceiver.class);
+
+   private static final String SPAT_START_FLAG = "0013"; // these bytes indicate
+                                                        // start of SPAT payload
+   private static final int HEADER_MINIMUM_SIZE = 20; // WSMP headers are at
```

Comment:
Which header size is this referring to? This is the spat header that we are currently using. https://github.com/usdot-jpo-ode/jpo-ode/blob/bfa632ed8a2f22578eac7b7d1421755a551b0bb4/data/wydotLogRecords.h#L107.

Reply:
To be honest I am not certain, this part of the code was based off a really old UDP receiver that was implemented into the ODE for BSM 3 years ago but it was deprecated and I revived it and updated the code at the end of last year. This might not even get used to be honest. I haven't seen WSMP messages get sent over UDP with a header most of the time.

Reply:
okay, sounds good. Thanks!